README.md

```diff
@@ -1,7 +1,528 @@
+    credentialSchema: {
+      id: 'https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json/KYCAgeCredential-v3.json',
+      type: 'JsonSchema2023'
+    },
+    proof: [ [Object] ]
+  }
+]
+{
+  id: 'http://localhost:3001/v1/did:iden3:tVE7RF655rj8Ws4rYCF2E32d46q575Gq2GHi2aKYf/claims/797c21b7-2197-11ee-9ce7-0242ac120005',
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    'https://schema.iden3.io/core/jsonld/iden3proofs.jsonld',
+    'https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld'
+  ],
+  type: [ 'VerifiableCredential', 'KYCAgeCredential' ],
+  expirationDate: '2030-07-12T17:08:07Z',
+  issuanceDate: '2023-07-13T16:08:15.702137458Z',
+  credentialSubject: {
+    birthday: 19960424,
+    documentType: 2,
+    id: 'did:iden3:tNdgTFxGLkPXhXtppLZzsX7YJmE88aMbkPZusa3zH',
+    type: 'KYCAgeCredential'
+  },
+  credentialStatus: {
+    id: 'http://localhost:3001/v1/did%3Aiden3%3AtVE7RF655rj8Ws4rYCF2E32d46q575Gq2GHi2aKYf/claims/revocation/status/3835285175',
+    revocationNonce: 3835285175,
+    type: 'SparseMerkleTreeProof'
+  },
+  issuer: 'did:iden3:tVE7RF655rj8Ws4rYCF2E32d46q575Gq2GHi2aKYf',
+  credentialSchema: {
+    id: 'https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json/KYCAgeCredential-v3.json',
+    type: 'JsonSchema2023'
+  },
+  proof: [
+    {
+      type: 'BJJSignature2021',
+      issuerData: [Object],
+      coreClaim: 'c9b2370371b7fa8b3dab2a5ba81b68382a000000000000000000000000000000010062fcf44584a9e517ad2c142519f533e1bc5e5b02335c43043b222fce0a00912170a9dcf64b58333fba5582097287445b126903b0dc90614831fcb1b59a040000000000000000000000000000000000000000000000000000000000000000b7ce99e40000000077e9d9710000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+      signature: '1d6cc275f7c9c1b937fb48d6476cc216a22cc981af640d2515fcb63471525f9d3a4ea667391dc2fa6f9faf45b5ceccef13a8c4166cd4d2db40b3166023472501'
+    }
+  ]
+}
+Inserting new entry to table Credentials
+Done!
+```
+
+### Create a challenge object
+
+Now we can use the verifiable credential obtained above, to create verifiable presentations for verifiers.
+
+A verifier first presents a challenge. Call the following endpoint on the verifier service to obtain the challenge object:
+
+```
+curl 'http://localhost:8000/api/v1/challenges' -H 'Content-Type: application/json' \
+--data-raw '{
+    "credentialSubject": {
+        "birthday": {
+            "$lt": 20021010
+        }
+    },
+    "context": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld",
+    "type": "KYCAgeCredential"
+}' | qrencode -o challenge.png
+```
+
+If you want to use selective disclosure, use the following endpoint instead:
```

Comment:
```suggestion
If you want to use selective disclosure, use the following payload instead (omitting the condition inside the `birthday` property value signals the verifier's desire for selective disclosure rather than zero knowledge proof):
```